tests/foreman/conftest.py

```diff
@@ -33,6 +38,64 @@ def worker_id(request):
         return 'master'
 
 
+@pytest.fixture(scope="session")
+def configured_settings():
```

Comment:
cannot be just `settings` ?

Reply:
I had some issues when naming it settings.